routers/web/user/oauth.go

```diff
@@ -185,6 +185,14 @@ func newAccessTokenResponse(grant *models.OAuth2Grant, clientSecret string) (*Ac
 				ErrorDescription: "cannot find application",
 			}
 		}
+		err = app.LoadUser()
+		if err != nil {
```

Comment:
It should probably return this user only on ErrNotFound otherwise it should return internal server error